src/main/java/uk/gov/indexer/dao/DestinationDBUpdateDAO.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.indexer.dao;
+
+import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.postgresql.util.PGobject;
+import org.skife.jdbi.v2.TransactionIsolationLevel;
+import org.skife.jdbi.v2.sqlobject.Bind;
+import org.skife.jdbi.v2.sqlobject.SqlQuery;
+import org.skife.jdbi.v2.sqlobject.SqlUpdate;
+import org.skife.jdbi.v2.sqlobject.Transaction;
+
+import java.nio.charset.StandardCharsets;
+import java.sql.SQLException;
+import java.util.List;
+
+public abstract class DestinationDBUpdateDAO implements DBQueryDAO {
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    static final String INDEXED_ENTRIES_TABLE = "ORDERED_ENTRY_INDEX";
+    static final String WATERMARK_TABLE = "STREAMED_ENTRIES";
+
+    @SqlUpdate("CREATE TABLE IF NOT EXISTS " + INDEXED_ENTRIES_TABLE + " (ID SERIAL PRIMARY KEY, ENTRY JSONB)")
```

Comment:
I'm not sure we should use SERIAL here -- might it be better if we explicitly specify the id of the entry from the source db instead?